msktutil.cpp

```diff
@@ -41,6 +41,11 @@
 
 int g_verbose = 0;
 
+/* Fatal error */
+void error_exit( const char *text) {
+    fprintf(stderr, "error_exit: %s: %s\n", text, strerror(errno));
```

Comment:
The user may not care what failed. But it helps us to diagnose failures. BTW, the message may be seen even before, but was implementation dependent, AFAIK